src/components/Dashboard/SortList/sortUtils.js

```diff
@@ -32,10 +32,9 @@ export const storage = {
   },
 
   set: (key, value) => {
-    let data = localStorage.getItem('SortList_Data') || '{}';
-    data = JSON.parse(data);
-    data[key] = value;
-    data = JSON.stringify(data);
-    localStorage.setItem('SortList_Data', data);
+    const data = localStorage.getItem('SortList_Data') || '{}';
+    const object = JSON.parse(data);
```

Comment:
What are `data` and `object`. Stop giving vague names to variables, and for once give it at least some thought

Reply:
data -> sortListDataRaw
object -> sortListData